src/Networks.tsx

```diff
@@ -55,10 +53,6 @@ export const defaultNetworkUrls: Record<NetworkName, {beaconApiUrl: string; elRp
     beaconApiUrl: process.env.REACT_APP_PRATER_BEACON_API || "https://lodestar-goerli.chainsafe.io",
     elRpcUrl: process.env.REACT_APP_PRATER_EXECUTION_API || "https://lodestar-goerlirpc.chainsafe.io",
   },
-  [NetworkName.ropsten]: {
```

Comment:
is Roptsten deprecated? I vaguely remember Phil requesting it should be supported. Perhaps touch base with him regarding the removal?